src/components/UserInput/UserInput.js

```diff
@@ -0,0 +1,59 @@
+import { useState } from 'react';
+import './UserInput.scss';
+
+const UserInput = ({
+  name,
+  koreanName,
+  condition,
+  inputValue,
+  loginInputHandler,
+}) => {
+  const [inputTitleGoUpBtn, setInputTitleGoUpBtn] = useState(false);
+
+  const inputGoUpEffector = () => {
+    if (inputValue.length > 0) return;
```

Comment:
```suggestion
    if (inputValue) return;
```
조금 더 세세하게 보자면 결국 inputValue.length가 0 이상이면
값이 입력되면 결국 true가 되는 것이기 때문에
이와같이 inputValue만 넣게 되더라도 true false로 사용이 가능합니다!
직접 적용해서 확인해보세요!

Reply:
네 적용하여 수정하였습니다. 불필요한 부분이 생각보다 많네요..! 피드백 감사드립니다 !